crates/rolldown_sourcemap/src/lib.rs

```diff
@@ -75,10 +68,7 @@ pub fn collapse_sourcemaps(sourcemap_chain: &[&SourceMap]) -> SourceMap {
               })
               .copied()
           }),
-          original_token
-            .get_name_id()
-            .and_then(|name_id| first_map.get_name(name_id))
-            .and_then(|name| names_map.get(name).copied()),
```

Comment:
`names_map` is a HashMap that `name -> idx`, which is a reverseMap of `first_map.names`, and `first_map.get_name` called `first_map.names.get` under the hood, so if you have a `original_idx`, then called 
`first_map.name.get(original_idx)`, then use the result `name` to get the idx, which is equivalent to the `original_idx`